lib/generators/rodauth/templates/app/test/fixtures/accounts.yml

```diff
@@ -0,0 +1,32 @@
+---
+# For both users: password is "password" for both accounts
+# This allows you do run request specs, or ActionDispatch::IntegrationTest
+# Here is an example:
+##### test_helper.rb
+#
+# class ActionDispatch::IntegrationTest
+#
+#  def login(account)
+#     #  you can login like this:
+#     # session[:account_id] = account.id
+#     # session[:authenticated_by] = ["password"] # or ["password", "totp"] for MFA
+#     # post login_path
+#     # or like this: 
+#     post "/login", params: {"login"=>"brian@queen.com", "password"=>"password"}
+#   end
+#
+#   def logout
+#     session.clear
+#     follow_redirect!
+#   end  
+# end
```

Comment:
I was under the impression that fixtures were available for any `ActiveSupport::TestCase` subclass, not just request and integration tests. But I don't think this whole segment makes sense here, because there is nothing fixture-specific here, and it will easily get outdated as request testing helpers improve (better to just consult the online docs).

Reply:
Agreed. I have removed this entire block.